docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -97,6 +97,19 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
 
 .. _list_variables:
 
+Boolean variables
+==============
+
+Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False``, and so on.
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
```

Comment:
Do the all the examples really use true/false? :thinking:

Reply:
@samccann could have more information about that

Reply:
It's a bit of a gray area. On the RST files we control, we hope to get most examples using true/false. For examples within collections, we can only suggest they follow that standard but we can't force it.